conf_dir/conf.template

```diff
@@ -0,0 +1,4 @@
+bind_port = 3000
+lnd_address = "tcp://localhost:10009"
```

Comment:
I will have to study what `tcp:` does, probably unsecure connection. Please use https instead or explain why you used `tcp`